core/src/test/scala/org/broadinstitute/dsde/rawls/model/ExecutionModelSpec.scala

```diff
@@ -126,6 +126,7 @@ class ExecutionModelSpec extends AnyFlatSpec with Matchers {
   "ExecutionServiceWorkflowOptions" should "serialize/deserialize to/from JSON" in {
     val test = ExecutionServiceWorkflowOptions(
       jes_gcs_root = "jes_gcs_root",
+      gcp_batch_gcs_root = "gcp_batch_gcs_root",
```

Comment:
I am not a fan of key/value ambiguity, even for tests.

```suggestion
      gcp_batch_gcs_root = "example_gcp_batch_gcs_root",
```

Reply:
Updated!